notebooks/00_quick_start/sar_movielens.ipynb

```diff
@@ -59,9 +59,7 @@
     "import sys\n",
```

Comment:
<p>oh I didn't understand how by removing the columns you would get `UserId MovieId Rating Timestamp` since they are not the default columns. I think you didn't rerun the notebook, right?</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/617/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
yes, my bad. fixing that